Source/Blazorise/wwwroot/colorPicker.js

```diff
@@ -36,10 +36,10 @@ export function initialize(dotnetAdapter, element, elementId, options) {
                 hsla: false,
                 hsva: false,
                 cmyk: false,
-                input: true,
+                input: options.showInputField === true ? true : false,
                 save: false,
-                clear: options.showClearButton || true,
-                cancel: options.showCancelButton || true
+                clear: options.showClearButton === true ? true : false,
+                cancel: options.showCancelButton === true ? true : false
```

Comment:
Can we go over what the previous logic did and what this one does now.

Previously:
`options.showClearButton || true`
Which one of these is right for this truth expression?
1 - if null or false returns true?
2 - if null returns true?
I admit I don't remember how js checks it like this. I'm guessing it's number 2 as number 1 does not really make sense, right?

`options.showClearButton === true`
Basically only if true, and excludes **"true"**, right?

Reply:
Basically the same explanation as above.